src/helpers/headers.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
I think this file should be called `/src/support/headers/index.json`. Notice the `.json` - there is no logic in this file, so just make it json.